web/pdf_viewer.js

```diff
@@ -872,6 +972,10 @@ class PDFViewer {
         if (this.defaultRenderingQueue) {
           this.update();
         }
+
+        if (this.#hiddenCopyElement) {
+          document.addEventListener("copy", this.#copyCallbackBound);
+        }
```

Comment:
Sorry, but isn't this still too early since no page has rendered yet?

I think that the "correct" spot would be *after* this code instead: https://github.com/mozilla/pdf.js/pull/16286/files#diff-4238fc7e96d72f4095859850f3d24a9298ec2dc8bd96437214a02dccbd48385cR905-R914

Reply:
I just picked up the wrong `}`. Thank you for noticing that.